lib/settings/utils.js

```diff
@@ -0,0 +1,78 @@
+var Promise     = require("bluebird")  
+  , coroutine   = Promise.coroutine
+  , Slideshow   = db.model('Slideshow')
+  , Exercise    = db.model('Exercise')
+  , _           = require('lodash')
+  
+
+var getConf = coroutine(function* getConf(conf) {
+  var slideConf = [];
+  for (var key in conf) {
```

Comment:
`Object.keys(conf).forEach` will help you avoid `hasOwnProperty`. Use this everywhere you want to enumerate the keys of an object.